src/main/kotlin/io/emeraldpay/dshackle/upstream/calls/DefaultEthereumMethods.kt

```diff
@@ -119,6 +119,9 @@ class DefaultEthereumMethods(
                     Chain.ETHEREUM_CLASSIC == chain -> {
                         "\"1\""
                     }
+                    Chain.POLYGON == chain -> {
+                        "\"1\""
```

Comment:
I believe this should be `"\"137\""` instead:

```
> net.version
"137"
```

Reply:
This is the result for eth_chainId, not net_version which is set elsewhere.

Reply:
Wait nevermind, you're right. I thought this was weird when I did it, and I remembered incorrectly. This is net_version and I do get 137 now